content/docs/admission-control/update.txt

```diff
@@ -0,0 +1,13 @@
+title KUDO Request Life Cycle
```

Comment:
I'd suggest naming this file `admission-instance-update.txt` to match the `png`.